src/libraries/Directory.Build.targets

```diff
@@ -64,6 +71,7 @@
 
     <BinPlaceRef Condition="'$(BinPlaceRef)' == '' and ('$(IsReferenceAssembly)' == 'true' or '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRef>
     <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' and ('$(IsRuntimeAssembly)' == 'true' or '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRuntime>
+    <EnableBinplacing Condition="'$(IsAspNetCoreAppAnalyzer)' == 'true' or '$(IsNetCoreAppAnalyzer)' == 'true'">true</EnableBinplacing>
```

Comment:
If you would add support for binplacing analyzers as suggested below you would also not need to set this property manually as binplacing would take care of it here: https://github.com/dotnet/arcade/blob/6b9d24236d8d1906284e6cb6c28e3fe93a69b7d2/src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/src/build/BinPlace.targets#L6

Reply:
We would need some property like the above two, I’m not seeing that as any more beneficial. We don’t need a “proper name” support here. We can leverage binplacing generically.

Reply:
I could remove this property entirely by modifying this condition to also handle `'@(BinPlaceDir)' != ''` https://github.com/dotnet/arcade/blob/6b9d24236d8d1906284e6cb6c28e3fe93a69b7d2/src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/src/build/BinPlace.targets#L12

Reply:
Yes that would be probably a better solution. I don't really like us setting `EnableBinplacing` here.